nvm.sh

```diff
@@ -2317,6 +2370,36 @@ nvm_npmrc_bad_news_bears() {
   return 1
 }
 
+# args: path to check and transform if needed
+nvm_ensure_posix_path() {
+  [ -z "$1" ] && nvm_err "Need the location as first parameter" && return 1
```

Comment:
```suggestion
  [ -z "${1-}" ] && nvm_err "Need the location as first parameter" && return 1
```

Reply:
Sure, I am not used to make POSIX compliant scripts, I am learning a lot working on nvm :D